fvm/transactionInvoker.go

```diff
@@ -289,7 +289,7 @@ func valueDeclarations(ctx *Context, env Environment) []runtime.ValueDeclaration
 			Kind:           common.DeclarationKindFunction,
 			IsConstant:     true,
 			ArgumentLabels: nil,
-			Value: interpreter.NewHostFunctionValue(
+			Value: interpreter.NewUnmeteredHostFunctionValue(
```

Comment:
I'm not sure about this @janezpodhostnik

Reply:
IMO this is fine. Its only one un-metered function, and we know what this function is supposed to do.